dev/com.ibm.ws.microprofile.config_fat/bnd.bnd

```diff
@@ -0,0 +1,58 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
+	test-applications/earlib.jar/src,\
+	test-applications/defaultSources.war/src,\
+	test-applications/types.war/src,\
+	test-applications/dynamicSources.war/src,\
+	test-applications/warVisibility.war/src,\
+	test-applications/stress.war/src,\
+	test-applications/sharedLibUser.jar/src,\
+	test-applications/customSources.war/src,\
+	test-applications/classLoaders.war/src,\
+	test-applications/converters.war/src,\
+	test-applications/testAppUtils.jar/src,\
+	test-applications/sharedLib.jar/src,\
+	test-applications/defaultSources.jar/src,\
+	test-applications/cdiConfig.war/src,\
+	test-applications/sharedLibUser.war/src,\
+	test-applications/brokenCDIConfig.war/src,\
+	test-applications/ordForDefaults.war/src,\
+	test-applications/cdiConfig.jar/src
+
+javac.source: 1.8
+javac.target: 1.8
+	
+test.project: true
+
+-buildpath: \
```

Comment:
Many of these dependencies are not needed or will be auto added because the this bundle contains "_fat"

The -buildpath can be simplified to:
```
-buildpath: \
	com.ibm.websphere.javaee.ejb.3.2;version=latest,\
	com.ibm.websphere.javaee.servlet.3.1;version=latest,\
	com.ibm.websphere.org.eclipse.microprofile.config.1.1;version=latest,\
	com.ibm.websphere.javaee.annotation.1.2;version=latest,\
	com.ibm.ws.security;version=latest,\
	com.ibm.websphere.security;version=latest,\
	com.ibm.websphere.javaee.concurrent.1.0;version=latest,\
	com.ibm.websphere.javaee.servlet.3.1;version=latest,\
	com.ibm.websphere.javaee.cdi.1.2;version=latest,\
    com.ibm.ws.org.apache.commons.logging.1.0.3;version=latest
```

Reply:
Thanks. Made the change.